lib/src/ssl_funcs/Md5.cc

```diff
@@ -229,8 +228,8 @@ std::string Md5Encode::GetHexStr(unsigned int num_str)
 }
 
 // function: Encode
-// @param src_info:要加密的信息
-// return :加密后的MD5值
+// @param src_info: the string to be encoded.
+// return : the string after encoding
 std::string Md5Encode::encode(const std::string &src_info)
```

Comment:
encode -> Encode ?

Reply:
As you can see, this part is copied from the Internet. It's simple and independent, so I just added some interfaces and made very limited changes. Also, if the OpenSSL lib is installed, it is not used. Anyway, thank you for your reminder, I will do better in the future.